### DISCUSSION TITLE
Jellyfin 10.9 Is amazing

### DISCUSSION THREAD

**Discussion by mhzawadi** [jellyfin/jellyfin#11746] [General]
I would just like to thank all the devs for your hard work on the update to 10.9, I have just skipped forward and dont have any issues with Jellyfin. 

I used to get all sorts of issues when skipping forward, out of sync audio or a complete fail of the video I was watching

**Comment by Alliyance** [comment]


**Reply by mhzawadi** [reply]
oh, what kind of issues do you get?

**Reply by mhzawadi** [reply]
I did follow the `Hardware Acceleration` from the docs and did some reading up on what my hardware can do, this page is a good start -> https://jellyfin.org/docs/general/administration/hardware-acceleration/

As I have a nuc like thing (Intel(R) Core(TM) i3-7100U CPU @ 2.40GHz) it just needed some extra config to make run buttery smoth

**Reply by Alliyance** [reply]
Well my biggest issue was fix today but the majority of the time, when there's a new update, something else isn't working. Like for exemple, the forced subs were activated if I choose Smart in users, now with the new update, you have to set this to Default. AAC audio was direct play, since the update, some users gets transcoded for unknown reasons. It's those little things that makes it a hard to keep up.
